HW_9/task_2.py

```diff
@@ -0,0 +1,56 @@
+class Employee:
+
+    def __init__(self, full_name: str, age: int, position: str, salary: float):
+        self.full_name = full_name
+        self.age = age
+        self.position = position
+        self._salary = salary
```

Comment:
same here. only private or protected variables. 

private is self.__var_name
protected: self._var_name